app/src/main/java/com/example/vlad/financemanager/data/models/Category.kt

```diff
@@ -0,0 +1,12 @@
+package com.example.vlad.financemanager.data.models
+
+import java.io.Serializable
+
+data class Category(var id: Int,
+                    var name: String,
+                    var icon: Int,
+                    var isCustom: Boolean,
+                    var isInputCategory: Boolean) : Serializable {
```

Comment:
Use https://proandroiddev.com/parcelable-in-kotlin-here-comes-parcelize-b998d5a5fcac